changelogs/fragments/final_countdown.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
```

```suggestion
major_changes:
```
Calling that "minor" is quite an under-statement ;-)

---

OK.